tests/pending/pos/i2797/Fork_1.java

```diff
@@ -0,0 +1,7 @@
+// Test is pending because we have no good way to test it.
```

Comment:
I've fixed the test. The issue was that the annotation needed to be called `Fork_1` since the name of the file is `Fork_1.java`. The javac error messages are eaten by our test suite, there's a WIP PR to fix that at https://github.com/lampepfl/dotty/pull/2811